auth/src/utils/getUnauthenticatedTenants.ts

```diff
@@ -13,9 +13,12 @@ export async function getUnauthenticatedTenants(subscriptionProvider: AzureSubsc
     const tenants = await subscriptionProvider.getTenants();
     const unauthenticatedTenants: TenantIdDescription[] = [];
     for await (const tenant of tenants) {
-        if (!await subscriptionProvider.isSignedIn(tenant.tenantId)) {
-            unauthenticatedTenants.push(tenant);
+        if (tenant) {
```

Comment:
This if statement is weird to me but I understand why you included it. Maybe add a comment for future us so we're not confused about why it's there and accidentally remove it.